virt-clone: add page (#5800)
tldr-pages/tldr: pages/common/virt-clone.md

# virt-clone

> Clone a libvirt virtual machine.
> More information: <https://manned.org/virt-clone>.

> - Clone a virtual machine and automatically generate a new virtual machine name, storage path, and MAC address:

Reviewer: ```suggestion
- Clone a virtual machine and automatically generate a new name, storage path, and MAC address:
```

`virt-clone --original {{vm_name}} --auto-clone`

> - Clone a virtual machine and specify the new virtual machine name, storage path, and MAC address:

Reviewer: ```suggestion
- Clone a virtual machine and specify the new name, storage path, and MAC address:
```

# virt-clone

> Clone a libvirt virtual machine.

> > More information: <https://virt-manager.org/>.

Reviewer: This link is really not a good one, because the site is very lacking in accessible information, as can be seen [here](https://github.com/tldr-pages/tldr/issues/5197#issue-795578123).  The man page, on the other hand, does provide a lot of nice detail.
```suggestion
> More information: <https://manned.org/virt-clone>.
```
(As we've discussed before, using the man page for everything is not the best idea, but in this case I do believe it's an improvement over this current link.)

Author: I've made this change in my latest commit. 

This was the link used in the virt-install tldr-page. I'll stop assuming that the More Information links in existing pages are necessarily correct.

> - Clone a virtual machine generating a new virtual machine name and storage path from the original name and path, and a random MAC address:

Reviewer 2: ```suggestion
- Clone a virtual machine with a random MAC address:
```
> generating a new virtual machine name and storage path from the original name and path

Isn't that already clear, since this is what cloning does? Please correct me if I'm wrong.

Author: I think I've addressed this in my latest commit.

> `virt-clone --original {{vm_name}} --auto-clone --mac RANDOM`

Reviewer: The man page reads this about `--mac` (bold mine):

>**If this parameter is omitted**, **or** the value `RANDOM` is specified a suitable address will be randomly generated.

so what do you think about this?

```suggestion
`virt-clone --original {{vm_name}} --auto-clone`
```

Author: I think I've addressed this in the new commit.
